hack/install.sh

```diff
@@ -11,13 +11,59 @@
 VERSION=${DAYTONA_SERVER_VERSION:-"latest"}
 BASE_URL=${DAYTONA_SERVER_DOWNLOAD_URL:-"https://download.daytona.io/daytona"}
 DESTINATION=${DAYTONA_PATH:-"/usr/local/bin"}
+CONFIRM_FLAG=false
+
+# Check for the -y flag
+for arg in "$@"; do
+  case $arg in
+    -y)
+    CONFIRM_FLAG=true
+    shift
+    ;;
+  esac
+done
 
 # Print error message to stderr and exit
 err() {
   echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&2
   exit 1
 }
 
+# Stop the Daytona server if it is running
+stop_daytona_server() {
+  if pgrep -x "daytona" > /dev/null; then
+    if [ "$CONFIRM_FLAG" = false ]; then
+      read -p "Daytona server is running. Do you want to stop it? (yes/no): " user_input
+      case $user_input in
+        [Yy][Ee][Ss] )
+          CONFIRM_FLAG=true
+          ;;
+        [Nn][Oo] )
+          echo "Please stop the Daytona server manually and rerun the script."
+          exit 0
+          ;;
+        * )
+          echo "Invalid input. Please enter 'yes' or 'no'."
+          exit 1
+          ;;
+      esac
+    fi
+
+    if [ "$CONFIRM_FLAG" = true ]; then
+      echo "Attempting to stop the Daytona server..."
+      if daytona server stop; then
+        echo "Daytona server stopped."
+      else
+        echo "Attempting to kill the process..."
+        pkill -x "daytona" || echo "Failed to stop Daytona server"
+      fi
+    fi
+  else
```

Comment:
No need for the `else`. This just might confuse users that are installing daytona for the first time.

Reply:
Updated!

Reply:
![Screenshot 2024-06-06 at 10 18 37](https://github.com/daytonaio/daytona/assets/26512078/c16c89f5-4e48-4ffb-a817-e0a00385b927)

@ezhil56x I can still see the `else` with the `echo`. Maybe you didn't ammend the commit.

Reply:
Missed the last else block. Ready to go now!